tables/ref_isolate_pairs/chen23-pair.csv

```diff
@@ -0,0 +1,6 @@
+ref_name,control_iso_name,iso_name
+Chen23,Omicron/BA.1 full genome,Omicron/BA.2 full genome
+Chen23,Omicron/BA.1 full genome,Omicron/BA.4 full genome
+Chen23,Omicron/BA.1 full genome,Omicron/BA.5 full genome
+Chen23,Omicron/BA.1 full genome,BA.2.75 full genome
+Chen23,Omicron/BA.1 full genome,BA.2.76 full genome
```

Comment:
@zachandfox please also use `Omicron/BA.2 full genome` as the control_iso_name to create 5 more pairs. The reason the validation program crashed was because `BA2_inf_20` was infected by BA.2